first game/textUI.py

```diff
@@ -11,14 +11,12 @@ def init():
 def drawText(text, font, text_col, x,y):
     img = font.render(text, True, text_col)
     #Dit zorgt ervoor dat de x en y coordinaten die je geeft altijd het center is
-    x = x - int(img.get_width() / 2)
+    x = x - img.get_width() / 2
     y = y - img.get_height() / 2
     settings.screen.blit(img, (x,y))
-    return int(img.get_width() / 2)

```

Comment:
Dat return int shit moest niet weggehaald worden

Reply:
Is het mogelijk om dat te veranderen of moeten we dat manually doen nadat we het geinstalleerd hebben?

Reply:
Ah ik heb het nu aangepast dus nu is het geoed